challenges-webapp/src/main/webapp/vue-app/challenges/components/ChallengeCard.vue

```diff
@@ -173,23 +169,27 @@ export default {
   },
   created() {
     this.getListWinners();
+    document.addEventListener('announcement-added', this.announcementAdded);
```

Comment:
whether use `$root.$on` everywhere to allow events to be catched/triggered inside the application only, or use `document.addEventListener` when it's a cross applications components. In this specific case, it's better to use `$root.$on` because, this will not be catched outside the application